<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="0" failed="2" skipped="0">
  <reporter-output>
    <line>
      <![CDATA[launching browser]]>
    </line>
    <line>
      <![CDATA[waiting]]>
    </line>
    <line>
      <![CDATA[clicking on gender button]]>
    </line>
    <line>
      <![CDATA[clicking on next button]]>
    </line>
    <line>
      <![CDATA[handling age drop down]]>
    </line>
    <line>
      <![CDATA[clicking on next button]]>
    </line>
    <line>
      <![CDATA[entering pin code]]>
    </line>
    <line>
      <![CDATA[entering mobile number]]>
    </line>
    <line>
      <![CDATA[Clicking on next button]]>
    </line>
    <line>
      <![CDATA[validateBannersTC execution started]]>
    </line>
    <line>
      <![CDATA[TC validateBannersis failed]]>
    </line>
    <line>
      <![CDATA[taking screenshot]]>
    </line>
    <line>
      <![CDATA[screenshot saving at location C:\Users\tussh\eclipse-workspace\CoverFoxTest\screenShots\validateBanners20240809203953.png]]>
    </line>
    <line>
      <![CDATA[validatePresenceOfSortButtonTC execution started]]>
    </line>
    <line>
      <![CDATA[TC validatePresenceOfSortButtonis failed]]>
    </line>
    <line>
      <![CDATA[taking screenshot]]>
    </line>
    <line>
      <![CDATA[screenshot saving at location C:\Users\tussh\eclipse-workspace\CoverFoxTest\screenShots\validatePresenceOfSortButton20240809203953.png]]>
    </line>
    <line>
      <![CDATA[closing browser]]>
    </line>
  </reporter-output>
  <suite started-at="2024-08-09T20:39:39 IST" name="Suite" finished-at="2024-08-09T20:39:55 IST" duration-ms="15795">
    <groups>
    </groups>
    <test started-at="2024-08-09T20:39:39 IST" name="Test" finished-at="2024-08-09T20:39:55 IST" duration-ms="15795">
      <class name="coverFoxTest.CoverFoxTC">
        <test-method is-config="true" signature="launchBrowser()[pri:0, instance:coverFoxTest.CoverFoxTC@15de0b3c]" started-at="2024-08-09T20:39:39 IST" name="launchBrowser" finished-at="2024-08-09T20:39:46 IST" duration-ms="6804" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[launching browser]]>
            </line>
            <line>
              <![CDATA[waiting]]>
            </line>
          </reporter-output>
        </test-method> <!-- launchBrowser -->
        <test-method is-config="true" signature="preConditions()[pri:0, instance:coverFoxTest.CoverFoxTC@15de0b3c]" started-at="2024-08-09T20:39:46 IST" name="preConditions" finished-at="2024-08-09T20:39:53 IST" duration-ms="6826" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[clicking on gender button]]>
            </line>
            <line>
              <![CDATA[clicking on next button]]>
            </line>
            <line>
              <![CDATA[handling age drop down]]>
            </line>
            <line>
              <![CDATA[clicking on next button]]>
            </line>
            <line>
              <![CDATA[entering pin code]]>
            </line>
            <line>
              <![CDATA[entering mobile number]]>
            </line>
            <line>
              <![CDATA[Clicking on next button]]>
            </line>
          </reporter-output>
        </test-method> <!-- preConditions -->
        <test-method signature="validateBanners()[pri:0, instance:coverFoxTest.CoverFoxTC@15de0b3c]" started-at="2024-08-09T20:39:53 IST" name="validateBanners" finished-at="2024-08-09T20:39:53 IST" duration-ms="18" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.fail(Assert.java:116)
at coverFoxTest.CoverFoxTC.validateBanners(CoverFoxTC.java:61)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[validateBannersTC execution started]]>
            </line>
            <line>
              <![CDATA[TC validateBannersis failed]]>
            </line>
            <line>
              <![CDATA[taking screenshot]]>
            </line>
            <line>
              <![CDATA[screenshot saving at location C:\Users\tussh\eclipse-workspace\CoverFoxTest\screenShots\validateBanners20240809203953.png]]>
            </line>
          </reporter-output>
        </test-method> <!-- validateBanners -->
        <test-method signature="validatePresenceOfSortButton()[pri:0, instance:coverFoxTest.CoverFoxTC@15de0b3c]" started-at="2024-08-09T20:39:53 IST" name="validatePresenceOfSortButton" finished-at="2024-08-09T20:39:53 IST" duration-ms="10" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.fail(Assert.java:116)
at coverFoxTest.CoverFoxTC.validatePresenceOfSortButton(CoverFoxTC.java:74)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[validatePresenceOfSortButtonTC execution started]]>
            </line>
            <line>
              <![CDATA[TC validatePresenceOfSortButtonis failed]]>
            </line>
            <line>
              <![CDATA[taking screenshot]]>
            </line>
            <line>
              <![CDATA[screenshot saving at location C:\Users\tussh\eclipse-workspace\CoverFoxTest\screenShots\validatePresenceOfSortButton20240809203953.png]]>
            </line>
          </reporter-output>
        </test-method> <!-- validatePresenceOfSortButton -->
        <test-method is-config="true" signature="closeBrowser()[pri:0, instance:coverFoxTest.CoverFoxTC@15de0b3c]" started-at="2024-08-09T20:39:53 IST" name="closeBrowser" finished-at="2024-08-09T20:39:55 IST" duration-ms="1471" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[closing browser]]>
            </line>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
      </class> <!-- coverFoxTest.CoverFoxTC -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
